Twittimation/RandomLikeAutomatonData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace Twittimation
+{
+    public class RandomLikeAutomatonData
+    {
+        public int MaxLikesPerFolloweePerDay { get; set; } = 0;
+        public int PercentageLikeChance { get; set; } = 10;
+        public Dictionary<long, int> LikesGivenPerFollowee { get; set; } = new Dictionary<long, int>();
```

Comment:
How does this know which date a like was given on?

Reply:
It resets if the current day is greater then LastUpdatedTimestamp, so therefore it knows it must have been given today.

Reply:
`LastUpdatedTimestamp` sounds like the last time the automaton was run. If it's run at 2pm, and then 3pm, and then 4pm...etc. How does it know when a "new day" has begun?

Reply:
It checks if the date in UTC is different:
`if (d.LastUpdatedTimestamp < now - now % (SecondsPerHour * 24))`